docs/DeveloperGuide.md

```diff
@@ -247,6 +247,61 @@ should end at destroy marker (X) but due to a limitation of PlantUML, the lifeli
  
 Taking into consideration the extra coupling involved, Alternative 1 was chosen as the current design for add command access to the model.
 
+### Delete Command Feature
+
+A key functionality of DevEnable is the ability to delete projects, issues, and clients into the system. The command 
+word for deleting will be `project`, `issue`, or `client`, depending on which entity is being deleted.
+This is followed by the flag `-d`, representing a Delete command. Next, it is followed by a compulsory argument to 
+initialise the entity.
+When a user enters a valid Delete command in the interface, `AddressBookParser#parseCommand` will be called which 
+processes the inputs, creates an instance of a command parser, and calls the `ProjectCommandParser#parse`,
+`IssueCommandParser#parse` or `ClientCommandParser#parse` method, depending on which entity is being added. Within 
+this method, the flag `-d` will be detected, calling `ProjectCommandParser#parseDeleteProjectCommand`,
+`IssueCommandParser#parseDeleteIssueCommand`, or `ClientCommandParser#parseDeleteClientCommand`, depending on which 
+entity is deleted, which checks for input argument validity with methods in `ParserUtil`.
+Finally, the parsed arguments are passed into and returned in an instance of the Delete Command entity and the 
+`DeleteProjectCommand#execute`, `DeleteIssueCommand#execute`, or `AddClientCommand#execute` is called depending
+on which entity is deleted, which retrieves the respective entity list from the system, deletes the entity from the 
+list to update it, and have the UI display the updated filtered entity list.
+
+#### Delete Project Command
+Compulsory prefix: p/<valid project id>
```

Comment:
should there be a compulsory prefix tag? because the prefix isn't used in the command